x-pack/plugins/security/server/saved_objects/secure_saved_objects_client_wrapper.test.ts

```diff
@@ -12,6 +12,18 @@ import { SavedObjectsClientContract } from 'kibana/server';
 import { SavedObjectActions } from '../authorization/actions/saved_object';
 import { AuditEvent, EventOutcome } from '../audit';
 
+jest.mock('../../../../../src/core/server/saved_objects/service/lib/utils', () => {
+  const { SavedObjectsUtils } = jest.requireActual(
+    '../../../../../src/core/server/saved_objects/service/lib/utils'
+  );
+  return {
+    SavedObjectsUtils: {
+      createEmptyFindResponse: SavedObjectsUtils.createEmptyFindResponse,
```

Comment:
Optional nit!
```suggestion
      ...SavedObjectsUtils,
```

Reply:
I started with that but it didn't work, jest didn't attach the correct mocks 😭🤷‍♀️

Reply:
Really? I thought I just tried it and it worked. I've experienced funky behavior regarding `requireActual` before though so I'm not surprised it gave you trouble.